File: JAVASCRIPT/DAY2/Task1.js
function ageChecker(sonsAge, fathersAge){

let ageOne = sonsAge *2;

let fathersAgeYearsAgo = fathersAge - ageOne;

return fathersAgeYearsAgo;

}


console.log("the fathers was " + ageChecker(13,50) + " older than his son");

File: PYTHON/DAY2/Task1.py
def age_checker(sonsAge, fathersAge ):
    ageOne = sonsAge *2;
    fathersAgeYearsAgo = fathersAge - ageOne;

    return fathersAgeYearsAgo;

result = age_checker(13, 50)
print(" the fathers is ",  result, " years older than his son ")
    